Automattic/jetpack
# Add new Licensing Flow in Jetpack Plugin

#### Changes proposed in this Pull Request:

* Create a new `js-package`, licensing, will handle the UI elements of activating and managing licenses in plugins
* Update main Jetpack Plugin to show the new `ActivationScreen` at `/license/activation`
* Update the licensing package API to expose `attach_license`

#### Jetpack product discussion



p1HpG7-cSg-p2

#### Does this pull request change what data or activity we track or use?



No

#### Testing instructions:






1. Boot branch and navigate to `wp-admin/admin.php?page=jetpack#/license/activation`
2. Verify the new license page appears
